libs/blocks/caas/caas.js

```diff
@@ -30,6 +32,30 @@ const loadCaas = async (a) => {
   a.insertAdjacentElement('afterend', block);
   a.remove();
 
+  const floodGateColor = getMetadata('floodgatecolor') || '';
+  if (floodGateColor === fgHeaderValue) {
+    state.fetchCardsFromFloodgateTree = true;
+  }
+
+  const { env } = getConfig();
+  const { host } = window.location;
+  let chimeraEndpoint = 'www.adobe.com/chimera-api/collection';
```

Comment:
The advanced panel allows users to select an endpoint:
![Screenshot 2023-09-22 at 1 18 14 PM](https://github.com/adobecom/milo/assets/9143715/03edb581-8cb6-4bbe-a9e8-2d680c3fb125)

This now makes it so that only the prod endpoint is used.   I think that's fine, but if that's what we want we should rip out the Advanced Endpoint option.

Reply:
I think we should keep the Advanced Endpoint option and not hard code it to prod. People would want to be able to point to stage content.

Reply:
@chrischrischris as part of this PR, we are using both Stage & Prod endpoints, as discussed the endpoints are base64 encoded. Hence we can keep the Stage & Prod both in the configurator, we have removed the Dev endpoint from the configurator.